### PR TITLE
Mois 38 id 005 request or delegate ownership

### PR DESCRIPTION
**Pull request by zlatax**
New features involving the leaving and deleting a household. The functions possible depend on the user's role in the household.

Member Role
- User can leave the house

Owner Role
- User can choose another member to become the owner of the household
- User can delete the household.
- (Owner can leave the house when they delegate their ownership)

### CODE REVIEW

File: app/pages/household/[houseId]/index.tsx [hunk 1]
```diff
@@ -112,23 +184,102 @@ const Household = () => {
...
+                            value="Delegate"
+                          ></input>
+                        </>
+                      ) : (
+                        <>
+                          <select
+                            name="newOwner"
+                            id="newOwner"
+                            className="select w-full max-w-xs"
+                            disabled
```
Comment (lolupp) [inline app/pages/household/[houseId]/index.tsx:241]: looks good but why are these disabled? Not able to choose someone to delegate to on preview

File: app/pages/household/[houseId]/index.tsx [hunk 2]
```diff
@@ -112,23 +184,102 @@ const Household = () => {
...
+              {isOwner() && (
+                <div className="flex flex-col">
+                  <form action="#" onSubmit={handleDelegateOwner}>
+                    <h3 className="text-xl font-bold">Delegate Ownership</h3>
+                    <p>
+                      Choose someone to handover the ownership of this UHouse!
+                    </p>
+                    <br />
+                    <div className="flex gap-x-5">
+                      {householdMembers && householdMembers.length > 0 ? (
```
Comment (lolupp) [inline app/pages/household/[houseId]/index.tsx:211]: not sure if this is working correctly. Doesnt show list of members on preview.

**Review by lolupp — COMMENTED** [review]
UI looks awesome but I wasnt able to choose another member to delegate from preview. Option was disabled even when there were multiple members.

**Review by geordanbennie — APPROVED** [review]
Lists users correctly, fixed previous version. Looks good!

**Pull request merged**